parsons/databases/redshift/constants.py

```diff
@@ -1,6 +1,6 @@
 COL_NAME_MAX_LEN = 120
 
-DECIMAL = 'decimal'
+DECIMAL = 'float'
```

Comment:
If we're going to go down this route, let's change the constant name to `FLOAT` as well.